app/views/shared/head/_lite_mode_css.erb

```diff
@@ -0,0 +1,148 @@
+
+<!-- Lite Mode Styles -->
+<style>
+  @charset "UTF-8";
+  body {
+    font-family: system, -apple-system, BlinkMacSystemFont, ‘Segoe UI’, Roboto, Oxygen, Ubuntu, Cantarell, ‘Open Sans’, ‘Helvetica Neue’, sans-serif;
+  }
+
+  body img {
+    display: none;
+  }
+
+  body#products .product {
+    margin-top: 1rem;
+    border-top: 1px solid #ddd;
+    padding-top: 1rem;
+  }
+
+  body#products .product .product-front > a {
+    display: none;
+  }
+
+  body#products .product > header {
+    display: none;
+  }
+
+  body#products .row#extra {
+    border-top: 1px solid #ddd;
+  }
+
+  .site-header {
+    margin-bottom: 15px;
+  }
+
+  .site-header nav {
+    margin-bottom: 15px;
+  }
+
+  ul.primary-navigation, ul.pagination {
+    display: inline-block;
+    margin: 0;
+  }
+
+  ul.primary-navigation li, ul.pagination li {
+    display: inline;
+  }
+
+  ul.primary-navigation {
+    padding: 0;
+  }
+
+  ul.primary-navigation li + li {
+    padding-left: 15px;
+  }
+
+  ul.pagination {
+    padding: 15px 0 0;
+  }
+
+  .secondary-navigation form {
+    margin-bottom: 15px;
+  }
+
+  .secondary-navigation .simple-field {
+    border-radius: 0;
+    font-weight: normal;
+    font-weight: 100;
+    font-size: 1rem;
+    padding: 10px;
+    width: 66%;
+    text-transform: uppercase;
+    border: 1px solid #ddd;
+  }
+
+  input[type="text"],
+  input[type="search"],
+  input[type="email"] {
+    border-radius: 0;
+    font-weight: normal;
+    font-weight: 100;
+    font-size: 1rem;
+    padding: 10px;
+    width: 66%;
+    text-transform: uppercase;
+    border: 1px solid #ddd;
+  }
+
+  input[type="email"],
+  #mce-MMERGE3 {
+    margin-bottom: 10px;
+  }
+
+  input[type="number"],
+  select {
+    border-radius: 0;
+    font-weight: normal;
+    font-size: 1rem;
+    padding: 5px;
+    border: 1px solid #ddd;
+  }
+
+  .actions button,
+  button[type="submit"],
+  input[name="commit"] {
+    font-size: .86rem;
+    padding: 5px;
+  }
+
+  .h-entry {
+    border-bottom: 1px solid #ddd;
+  }
+
+  .h-entry .categories ul {
+    display: inline-block;
+    margin-top: 0;
+    list-style: none;
+    padding-left: 15px;
+  }
+
+  .sr-only {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+  }
+
+  .screen-reader-only {
+    display: none;
+  }
+
+  .p-x-title {
+    margin: .5em 0;
+    font-size: 2em;
+  }
+
+  .p-x-subtitle {
+    margin-top: 0;
+    font-size: 1.25em;
+  }
+
+  img[src*="start-header"] {
+    display: none;
+  }
+</style>
```

Comment:
Please rename the file to `_lite_mode_css.html.erb` since it is technically a snippet of HTML because of the `<style>` tags.